coderd/workspaces_test.go

```diff
@@ -198,6 +198,49 @@ func TestAdminViewAllWorkspaces(t *testing.T) {
 	require.Equal(t, 0, len(memberViewWorkspaces.Workspaces), "member in other org should see 0 workspaces")
 }
 
+func TestWorkspacesSortOrder(t *testing.T) {
```

Comment:
Note: this test will fail unless the equivalent ordering is also implemented for the fake in-memory database.

Reply:
Implemented